kotlin-generator/src/main/scala/models/generator/kotlin/KotlinGenerator.scala

```diff
@@ -335,8 +335,13 @@ class KotlinGenerator
               response.code.asInstanceOf[ResponseCodeInt].value < 299
           })
 
+          var isUnitResponse = false
```

Comment:
Using `var` in scala should be avoided.  You can do this instead: ```          val isUnitResponse = maybeSuccessfulResponse.map(successfulResponse =>
            successfulResponse.`type` == Datatype.Primitive.Unit.name
          ).getOrElse(false)```

Reply:
Fixed, used the first way